### DISCUSSION TITLE
New roms doesn't show in NDS folders even after refresh list

### DISCUSSION THREAD

**Discussion by Hokuzhiroku** [OnionUI/Onion#1561] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

V.4.3.0

### Miyoo Firmware Version

202305052130

### Provide a clear and concise description of the issue

After updated to version 4.3, After add new roms to NDS folder, the new game doesn't show even after refresh list. 

### Steps to Reproduce

Add Drastic in previous version, and update to version 4.3

### Provide any additional context or information that may help us investigate your issue

My game that added before update it's still playable and fine.

**Comment by Aemiii91** [comment]
What's your exact contents of `Roms/NDS` ?

**Reply by Hokuzhiroku** [reply]
Some roms with .zip and .nds extension, it still works but when i added some nds files it still doesn't show up

The new added roms is tested with delta emu and works well

**Comment by Hokuzhiroku** [comment]
Problem solved
I deleted previous list file and all work again
